### PR TITLE
Part1 Pull Request

### PR DESCRIPTION
**Pull request by ESSpaulding**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -56,10 +62,44 @@ void variableDeclarations()
...
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
-    
-    
-    
+    int counter = 0; 
+    int people = 6;
+    int tries = 10;
+    bool flag = 0;
+    bool check = false;
+    bool signBit = 1;
```
Comment (matkatmusic) [inline main.cpp]: Never use 0 or 1 to represent `true` or `false`.
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -56,10 +62,44 @@ void variableDeclarations()
...
+    ignoreUnused(input);
+    ignoreUnused(initial);
+    ignoreUnused(alphabetSoup);
+    ignoreUnused(question);
+    ignoreUnused(peak);
+    ignoreUnused(positiveOnly);
+    ignoreUnused(rectifier);
+    ignoreUnused(timeLine);
+    ignoreUnused(blackHole);
+    ignoreUnused(antiGravity);
```
Comment (matkatmusic) [inline main.cpp]: pass all of your variables to a single call to ignoreUnused.
- [x] revise

File: main.cpp [hunk 3]
```diff
@@ -125,33 +203,43 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
     ignoreUnused(carRented);
+    ignoreUnused(averageScore);
+    ignoreUnused(coinFlip);
+    ignoreUnused(sumOfAllRolls);
+    ignoreUnused(fullWaveRectifier);
+    ignoreUnused(lastStreamIn);
+    ignoreUnused(itIsNoise);
+    ignoreUnused(increment);
+    ignoreUnused(typeOfMotor);
+    ignoreUnused(secretDecoder);
```
Comment (matkatmusic) [inline main.cpp]: pass all of your variables to a single call to ignoreUnused.
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 2 above)
Comment (ESSpaulding) [reply main.cpp]: I wish I could have written a program to put them in that function.  Woulda saved so much time!

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**